### PR TITLE
Add disk type examples to README.md

### PR DESCRIPTION
**Pull request by noahl**

### CODE REVIEW

**Comment by noahl** [comment]
@voellm please take a look.

File: README.md [hunk 1]
```diff
@@ -682,6 +683,19 @@ Example:
 ./pkb.py --cloud=GCP --zone=us-central1-a --benchmarks=iperf,ping
 ```
 
+The disk type names vary by provider, but the following table summarizes some
+useful ones.
+
+Cloud name | Network-attached SSD | Network-attached HDD
```
Comment (voellm) [inline README.md]: What about local ssd?

File: README.md [hunk 2]
```diff
@@ -682,6 +683,19 @@ Example:
...
+-----------|----------------------|---------------------
+GCP | pd-ssd | pd-standard
+AWS | gp2 | standard
+Azure | premium-storage | standard-disk
+
+Also note that `--data_disk_type=local` tells PKB not to allocate a separate
```
Comment (voellm) [inline README.md]: When there are 2 disks... remote HDD and local SSD like on some Azure instances how is this resolved?

File: README.md (hunk 1 above)
Comment (noahl) [inline README.md]: There's a note below the table about local SSD. The `local` disk type is special.

File: README.md (hunk 2 above)
Comment (noahl) [inline README.md]: I'll add a note about that.

**Comment by noahl** [comment]
@voellm I just updated. Please take another look.

**Comment by meteorfox** [comment]
@noahl Can you add Rackspace data disk types as well?

boot - uses the boot disk
local - uses local disks that is not the boot disk
cbs-sata - remotely attached HDD volume from Cloud Block Storage
cbs-ssd - remotely attached SSD volume from Cloud Block Storage

Thanks!

File: README.md (hunk 1 above)
Comment (voellm) [inline README.md]: Can you update for @meteorfox  comment ... Please add Rackspace.  Thanks!

boot - uses the boot disk
local - uses local disks that is not the boot disk
cbs-sata - remotely attached HDD volume from Cloud Block Storage
cbs-ssd - remotely attached SSD volume from Cloud Block Storage

**Comment by hildrum** [comment]
Would this be useful in wiki form:
https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/wiki/Disk-type-flags

**Comment by voellm** [comment]
very...

On Thu, Mar 10, 2016 at 12:27 PM, Kris Hildrum <notifications@github.com>
wrote:

> Would this be useful in wiki form:
>
> https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/wiki/Disk-type-flags
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/871#issuecomment-195033192>
> .
>



-- 
Anthony F. Voellm (aka Tony)
Google Voice:  (650) 516-7382
<https://www.google.com/voice/b/0?pli=1#phones>
Blog: http://perfguy.blogspot.com/
Benchmark like a pro... PerfKitBenchmarker
<https://github.com/GoogleCloudPlatform/PerfKitBenchmarker>

**Comment by noahl** [comment]
@meteorfox Sorry I didn't include Rackspace originally! I've added CBS-SSD and CBS-SATA. I should also make it clear that I don't intend for the table to be comprehensive. It's just a list of some commonly-used flag values.

**Comment by noahl** [comment]
@voellm I've added examples to the README, and Kris put them on the wiki. Okay to merge?

**Comment by hildrum** [comment]
LGTM.

**Pull request merged**